plugins/MosaicCamera/src/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+INCLUDE_DIRECTORIES(
+     .
+     gui
+     ${CMAKE_BINARY_DIR}/plugins/MosaicCamera/src
+     ${CMAKE_BINARY_DIR}/plugins/MosaicCamera/src/gui
+)
+
+LINK_DIRECTORIES(${BUILD_DIR}/src)
+
+SET(MosaicCamera_SRCS
+     MosaicCamera.hpp
+     MosaicCamera.cpp
+     MosaicTcpServer.hpp
+     MosaicTcpServer.cpp
+     gui/MosaicCameraDialog.hpp
+     gui/MosaicCameraDialog.cpp
+)
+
+SET(MosaicCameraDialog_UIS
+     gui/mosaicCameraDialog.ui
+)
+
+SET(MosaicCamera_RES ../resources/MosaicCamera.qrc)
+QT5_ADD_RESOURCES(MosaicCamera_RES_RCC ${MosaicCamera_RES})
```

Comment:
The plugin cannot be compiled with Qt6 - please fix it (see other plugins for an example)